integration_tests/macros/equality.sql

```diff
@@ -0,0 +1,96 @@
+{% test equality(model, compare_model, compare_columns=None, precision = None) %}
```

Comment:
Is it worth putting this in `snowplow_utils` properly within this release? So that other packages can also leverage this?

Reply:
I nearly did, but I'm hoping it will get added to dbt_utils, or at least they'll say that they aren't going to add it. Once they respond to the issue we can make a decision after that but leave it here for now?